### DISCUSSION TITLE
嘶～啊——

### DISCUSSION THREAD

**Discussion by C-Y-G-G** [share121/inter-knot#4514] [崩坏：星穹铁道]
![Image_1725651028817](https://github.com/user-attachments/assets/686fff40-4e71-470d-9cd1-97c4f76119ee)

**Comment by March-777th** [comment]
神金怎么变出生了<img src="https://bbs-static.miyoushe.com/static/2024/07/31/73346f2b98d45b62da9139d2774fa856_8641114165088315754.png" alt="三月七 告退" width="75" >

**Comment by basdaba** [comment]
你这个图片怎么会有声音
